test/parallel/test-cluster-worker-wait-server-close-schednone.js

```diff
@@ -0,0 +1,80 @@
+var common = require('../common');
+var assert = require('assert');
+var cluster = require('cluster');
+var net = require('net');
+
+if (cluster.isWorker) {
+
+  net.createServer(function(socket) {
+
+    // Wait for any data, then close connection
+    socket.on('data', socket.end.bind(socket));
+
+  }).listen(common.PORT, '127.0.0.1');
```

Comment:
Could you change the `127.0.0.1` to [`common.localhostIPv4`](https://github.com/iojs/io.js/blob/v1.x/test/common.js#L41) here and below please?